### PR TITLE
Story 41

### LINKED ISSUE #75: User Story 41, Admin visits Merchant Index Page
```
As an admin user
When I visit the merchant's index page at "/merchants"
I see all merchants in the system
Next to each merchant's name I see their city and state
The merchant's name is a link to their Merchant Dashboard at routes such as "/admin/merchants/5"
I see a "disable" button next to any merchants who are not yet disabled
I see an "enable" button next to any merchants whose accounts are disabled
```

DO THESE STORIES FIRST:
- User Story 39, Merchant Index Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #78

child of #125

depends on #77

### PR DESCRIPTION
**Pull request by cnodland**
closes #75

### CODE REVIEW

File: app/views/admin/merchants/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<h1>All Merchants in the system</h1>
+
+<% @merchants.each do |merchant| %>
+<section id="merchant-<%= merchant.id %>">
+  <p>Name: <%= link_to "#{merchant.name}", admin_merchant_path(merchant) %></p>
```
Comment (whois-jon-peterson) [inline app/views/admin/merchants/index.html.erb:5]: `merchant.name` probably doesn't need to be interpolated here.

File: config/routes.rb [hunk 2]
```diff
@@ -7,6 +7,9 @@
     get '/dashboard', to: "orders#index"
     resources :users, only: [:index, :show]
     get '/merchants/:id', to: "merchants#show", as: :merchant
+    get '/merchants', to: "merchants#index", as: :merchants
+    get '/merchant/:id/enable', to: "merchants#enable", as: :merchant_enable
```
Comment (wipegup) [inline config/routes.rb:11]: Can we make these more RESTful by making them `PATCH` calls to "merchants#update"?

Comment (wipegup) [reply config/routes.rb:11]: Looks like this was changed in your next PR

**Review by wipegup — APPROVED** [review]
